src/abstractdataframe/abstractdataframe.jl

```diff
@@ -970,15 +970,52 @@ julia> dropmissing(df, [:x, :y])
 @inline function dropmissing(df::AbstractDataFrame,
                              cols::Union{ColumnIndex, MultiColumnIndex}=:;
                              view::Bool=false, disallowmissing::Bool=!view)
+    # Identify Bool mask of which rows have no Missings
     rowidxs = completecases(df, cols)
     if view
         if disallowmissing
             throw(ArgumentError("disallowmissing=true is incompatible with view=true"))
         end
         return Base.view(df, rowidxs, :)
     else
-        newdf = df[rowidxs, :]
-        disallowmissing && disallowmissing!(newdf, cols)
+        # With exact indices we can skip a lot of iterations (as opposed to indexing via a mask)
```

Comment:
This is OK. Maybe add a comment that this especially makes sense because we assume that typically we keep most of rows (only few are dropped).

Reply:
I'm not sure that would justify it. It generally faster than indexing via a mask, but there is an overhead of the _findall call, so this approach is justified if:
- if you're dropping many rows (which can happen easily if you have missing across many columns... the % of rows to drop goes up very quickly with the "ANY missing")
- if you have many columns (which is extremely common)

So what really justifies the `_findall` is the fact that most users have multiple columns. I'll reflect in the note